code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -664,14 +657,18 @@
 	..()
 
 /datum/chemical_reaction/solidification
-	name = "Metal solidification"
```

Comment:
This base class conflicted with the `solidification/iron` and did nothing on its own